examples/viewer.py

```diff
@@ -80,6 +83,9 @@ def __init__(self, sim_settings: Dict[str, Any]) -> None:
         self.sim: habitat_sim.simulator.Simulator = None
         self.reconfigure_sim()
 
+        # compute NavMesh
+        self.navmesh_config_and_recompute()
```

Comment:
As @Skylion007 suggested:
```suggestion
        # compute NavMesh if not already loaded by the scene.
        if not self.sim.pathfinder.is_loaded:
            self.navmesh_config_and_recompute()
```

Reply:
You also need to set `self.navmesh_success ` to `self.sim.pathfinder.is_loaded`. In fact, I would replace that variable with a direct access to `self.sim.pathfinder.is_loaded` and only use the `recompute_navmesh` return value to throw an error / exception if it fails.

Reply:
Yeah, agreed. Let's just use `is_loaded` everywhere instead of tracking the recompute with a variable.